tests/dates.test.ts

```diff
@@ -12,19 +14,28 @@ import Romcal from '../lib';
  * @param    dayOfWeek - Day of week to count
  * @returns  Number of week days in the range, including the end date
  */
-const getNumberOfWeekDaysInRange = ({start, end, dayOfWeek = 0}: {start: Date, end: Date, dayOfWeek?: 0 | 1 | 2 | 3 | 4 | 5 | 6}): number => {
+const getNumberOfWeekDaysInRange = ({
+  start,
+  end,
+  dayOfWeek = 0,
+}: {
+  start: Date;
+  end: Date;
+  dayOfWeek?: 0 | 1 | 2 | 3 | 4 | 5 | 6;
+}): number => {
   // Calculate the number of weeks (even partial) from the first target day to the end
   return Math.ceil(
     // Calculate the number of days from the first target day to the end
     Math.max(
       // Calculate the number of days between start and end, including the end date (thus `+ 1` at the end)
-      Math.ceil((end.getTime() - start.getTime()) / 864e5) + 1
-      // Calculate the nb of days before the next target day (e.g. next Sunday after start)
-      - (7 + dayOfWeek - start.getUTCDay()) % 7,
-      0
-    ) / 7
+      Math.ceil((end.getTime() - start.getTime()) / 864e5) +
+        1 -
+        // Calculate the nb of days before the next target day (e.g. next Sunday after start)
+        ((7 + dayOfWeek - start.getUTCDay()) % 7),
+      0,
+    ) / 7,
```

Comment:
That lonely `1 -` on a line does not look nice IMHO.

Reply:
Prettier rules. It doesn't bother me more than that, especially since it's happening only here.